src/test/unit/requests/accountManagementRequests.test.ts

```diff
@@ -48,7 +48,7 @@ const cftIdamUserEndpoint = '/account/provenance/CFT_IDAM/';
 const updateAccountEndpoint = '/account/provenance/PI_AAD/';
 const getAllAccountsEndpoint = '/account/all';
 const getUserByUserIdEndpoint = '/account/';
-const deleteUserByUserIdEndpoint = '/account/delete/';
+const deleteUserByUserIdEndpoint = '/account/v2/';
```

Comment:
Could we add a test in here to check that the admin ID is passed through as a header?

Reply:
Update the unit test by adding the header to the stub